### ISSUE TITLE
Refused to load the script because it violates the following Content Security Policy directive:

### ISSUE THREAD

**Issue by nounder** [thetarnav/solid-devtools#311]
Chrome extension throws following error on every visited page.

```
Refused to load the script 'chrome-extension://6f7ca782-90d7-4398-aaf3-40ac955f69f9/assets/content.ts-14921b37.js' because it violates the following Content Security Policy directive: "script-src 'self' 'wasm-unsafe-eval' 'inline-speculation-rules'". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.
```

Pressing address bar icon shows: "Solid not detected" even after following getting started guide.

Chrome version:  130.0.6723.117

**Comment by elvstejd** [comment]
+1 
And duplicate of #310 I believe

**Comment by CaptainBruven** [comment]
+1

**Comment by kzimmer-taponit** [comment]
+1

**Comment by kevinvantus** [comment]
+1

**Comment by thetarnav** [comment]
I think this is the same as #310. *Should* be fixed now in 0.31.0.
